src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -85,6 +87,8 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
             _solutionEventMonitor = new SolutionEventMonitor(_workspace);
 
             TrackBulkFileOperations();
+
+            RegisterCodeCleanupProvider();
```

Comment:
@jinujoseph I guess this kind of thing we expect not to show up until roslyn is fully initalized?